firmware/integration/rusefi_config.txt

```diff
@@ -1209,9 +1249,6 @@ int16_t tps2Max;Full throttle#2. tpsMax value as 10 bit ADC value. Not Voltage!\
 	brain_pin_e binarySerialRxPin;	
 	
 	brain_pin_e[AUX_DIGITAL_VALVE_COUNT iterate] auxValves
-	switch_input_pin_e tcuUpshiftButtonPin
```

Comment:
you can't remove stuff from the middle of the config, it'll change the address of everything after it (which breaks compatibility)

Reply:
but it didn't change the address of everything after, because of the automatic 4-byte alignment

Reply:
I was brute force programming at that point, and removing everything was what worked.